src/test/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterControllerTest.java

```diff
@@ -85,22 +82,6 @@ public void should_return_connection_exception_when_service_fails_due_to_thread_
         verifyNoMoreInteractions(authService, letterService);
     }
 
-    @Test
-    public void should_return_400_bad_request_when_service_fails_to_serialize_letter() throws Exception {
-        given(authService.authenticate("auth-header-value")).willReturn("service-name");
-        willThrow(JsonProcessingException.class).given(letterService).send(any(LetterRequest.class), anyString());
-
-        sendLetter(readResource("letter.json"))
-            .andExpect(status().isBadRequest())
-            .andExpect(content().string(
-                containsString("Exception occurred while parsing letter contents")));
-
-
-        verify(authService).authenticate("auth-header-value");
-        verifyNoMoreInteractions(authService);
-
-    }
```

Comment:
is this related?

Reply:
this was related to service bus when we serialised the letter. not needed now